### PR TITLE
[DOCFIX] Improve Minio docs

### PR DESCRIPTION
**Pull request by ZacBlanco**
Minor improvements for style in Minio docs. Added a section on verifying the setup once the user has entered the configuration.

### CODE REVIEW

File: docs/en/ufs/Minio.md [hunk 1]
```diff
@@ -33,8 +33,8 @@ Minio is an object storage server built for cloud applications and DevOps. Minio
 source alternative to AWS S3.
 
 Launch a Minio server instance using the steps mentioned
-[here](http://docs.minio.io/docs/minio-quickstart-guide). Then create a bucket (or use an existing
-bucket). Once the server is launched, keep a note of Minio server endpoint, accessKey and secretKey.
+[here](http://docs.minio.io/docs/minio-quickstart-guide). Then either create or use an existing
```
Comment (calvinjia) [inline docs/en/ufs/Minio.md]: `create an existing bucket` doesn't make sense?

File: docs/en/ufs/Minio.md [hunk 2]
```diff
@@ -60,3 +60,23 @@ aws.secretKey=<MINIO_SECRET_KEY_ID>
...
+Verify Alluxio is running by navigating to [http://localhost:19999](http://localhost:19999) or by
+examining the logs to ensure the process is running.
+
+Then, to run tests using some basic Alluxio operations execute the following command:
+
+```bash
+$ bin/alluxio runTests
+```
+
+If there are no errors then Minio is configured properly!
```
Comment (calvinjia) [inline docs/en/ufs/Minio.md]: Are there any common errors we should call out?

File: docs/en/ufs/Minio.md (hunk 1 above)
Comment (ZacBlanco) [reply docs/en/ufs/Minio.md]: But the text reads "Then either create or use an existing bucket.."

Comment (calvinjia) [reply docs/en/ufs/Minio.md]: `either create a new bucket or use an existing bucket` would make sense to me.
The current wording implies `create` and `use` both act on `an existing bucket`

File: docs/en/ufs/Minio.md (hunk 2 above)
Comment (ZacBlanco) [reply docs/en/ufs/Minio.md]: Added a section for basic troubleshooting

File: docs/en/ufs/Minio.md (hunk 1 above)
Comment (ZacBlanco) [reply docs/en/ufs/Minio.md]: I've updated the wording

**Pull request merged**